go/protocol/extras.go

```diff
@@ -30,6 +30,7 @@ const (
 	SIG_ID_LEN         = 32
 	SIG_ID_SUFFIX      = 0x0f
 	SIG_SHORT_ID_BYTES = 27
+	SigIDQueryMin      = 16
```

Comment:
can we make this ~8 characters; or ~6 characters?  something more akin to short git hashes.